assets/src/scripts/d3-rendering/tooltips.js

```diff
@@ -7,7 +7,7 @@
  */
 
 // Required to initialize USWDS components after page load
-import components from '../../node_modules/uswds/src/js/components';
+import components from 'uswds/src/js/components';
```

Comment:
Is this an 'alias-plugin' thing, or just a standard Rollup thing?

Reply:
Just standard rollup. I'm not sure why I didn't try it this way before.